lib/collection/src/recommendations.rs

```diff
@@ -69,6 +70,7 @@ async fn retrieve_points(
                 ids,
                 with_payload: Some(WithPayloadInterface::Bool(false)),
                 with_vector: WithVector::Selector(vector_names),
+                read_consistency: ReadConsistency::Factor(1), // TODO: Which `ReadConsistency` should be used here!?
```

Comment:
See the `TODO`.

Should we propagate the read consistency value (from `recommend_batch_by`) used by the respective recommend request in the batch here?

Reply:
Decided to redesign requests/messages a bit. The question is resolved with the redesign.